source/fundamentals/monitoring.txt

```diff
@@ -0,0 +1,406 @@
+==========
+Monitoring
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+.. What is on this page?
+
+This guide explains how you can set up and configure **monitoring** in the
+MongoDB Java driver.
+
+.. What do any new terms mean?
+
+Monitoring is the process of getting information about the activities a piece of
+software performs. You can use the information you get from monitoring in your
+application or an application performance management library. 
+
+Monitoring the MongoDB Java driver lets you understand the
+driver's resource usage and performance, and can help you make informed
+decisions when designing and debugging your application.    
+
+.. What can you expect to see on this page?
+
+On this page you will learn how to perform these tasks:
```

Comment:
I don't think the "guide"/"page" discrepancy is a problem. Our sidebar TOC refers to all of our guides/pages as "pages" (says "On This Page" for as far as I can tell all of our pages), but we often refer to them as guides in the intro. A "guide" is necessarily a "page" within our docs.

I do not agree with the second edit. I believe the proposed rephrasing is less task oriented than the current phrasing on the page (I believe readers are interested in "tasks" they wish to perform rather than "sections" they may read).

Reply:
This edit suggests it may be beneficial to discuss how we would like to refer to our pages. Is "guide" the only appropriate term? Are "page" and "guide" both acceptable?